ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -121,6 +131,44 @@ const SidebarComponent: React.FC<RootComponentProps> = props => {
     };
   }, [routing]);
 
+  // const allApps = useMemo(() => {
+  //   return [...(worldApps || []), ...(userInstalledApps || [])];
+  // }, [worldApps, userInstalledApps]);
+
+  useEffect(() => {
+    if (loginQuery.data?.id) {
+      subtitleUi.current = (
+        <DidField
+          did={loginQuery.data?.id}
+          textColor="grey7"
+          copyLabel={t('Copy to clipboard')}
+          copiedLabel={t('Copied')}
+        />
+      );
+    } else {
+      subtitleUi.current = (
+        <Text variant="footnotes1" customStyle="text-grey5 whitespace-normal" truncate breakWord>
+          {t('Connect to see exclusive member only features.')}
+        </Text>
+      );
+    }
+  }, [isLoading, loginQuery.data?.id, t]);
+
+  useEffect(() => {
+    if (myProfileQuery.data?.profile?.did?.id) {
+      setProfileName(myProfileQuery.data?.profile?.name);
+    } else if (loginQuery.data?.id) {
+      setProfileName('Logged-in User');
```

Comment:
why this section had to be moved up? it makes the review way harder

Reply:
I moved some of the existing blocks around, that's why they got highlighted. the PR primarily introduced changes relating to the CTA card